the_hat_game/game.py

```diff
@@ -13,6 +13,7 @@
 import the_hat_game.nltk_setup  # noqa: F401
 from the_hat_game.loggers import c_handler, logger
 from the_hat_game.players import RemotePlayer
+from data.utils import corpus_to_words
```

Comment:
`data/` migrated to `server/` folder. The idea was to split the logic into two parts (the-hat-game part - only game logic, don't know anything about server implementation; server - everything about working with cloud, uploading texts, etc).
So maybe we should move some data functions to `the-hat-game/data.py`. Those which can be used in the-hat-game and aren't related to server directly. Namely, `sent_2_words`, `corpus_to_words`, and maybe `save_words` (in this case we should remove default `vocab_path` arg value, because it's related to server implementation).

Reply:
fixed in https://github.com/production-ml/the-hat-game/pull/24/commits/0481ab441bc6c77a95e00abb49d55c9507053672

`sent_2_words`,` corpus_to_words` were added, `save_words ` not (it is not used anywhere)